### ISSUE TITLE
ANY doesn't work when function has parameter in Go

### ISSUE THREAD

**Issue by fnsne** [xylo/intellij-postfix-templates#165]
In PyCharm and Goland, I add a postfix template below.
In PyCharm
```
.ae: assert equal
    ANY →   self.assertEqual($expected$, $expr$)
```
In the Goland
```
.ae: assert Equal
	ANY  →	assert.Equal($t$, $expr$, $expected$,"")
```
In PyCharm, I can use this template for function with any number of parameters.
input
```
A.show(a,b).ae
```
will get
```
self.assertEqual( | , A.show(a,b))
```
In the Goland, when I type `.ae` after a method with no empty parameter, the template won't show.
```
A.show(a, b).ae
```
But after extracting the method to a variable, it works.
```
actual := A.show(a, b)
actual.ae
```

it's that a bug of this plugin or the Goland IDE?
Thanks.

### Updated:
I don't know why, but after adding space before right paratheses, it works.

**Comment by xylo** [comment]
I guess it's a bug in the plugin.  The expression selector needs more fine-tuning.  However, I don't have the time to fix this.  PRs are welcome.

**Comment by fnsne** [comment]
I found that it works right when the last parameter is `" `(string), `}`(struct, slice, map ......).
And it always works with any parameters by adding a space before the method parentheses.
like below

it works.
```
A.show(strA ).ae
```

But this not works.
```
A.show(strA).ae
```

**Comment by fnsne** [comment]
I found the PSI structs of Goland is a little different from Idea.
That cause Goland cannot find the `A.show(strA)` expression.

So I copy `selectorAllExpressionsWithCurrentOffset` and `collectExpressions` from `SimpleStringBasedPostfixTemplate` to `CustomGoStringPostfixTemplate`

And remove some offset calculation sees unnecessary in Go.

I think it works great.

**Comment by xylo** [comment]
Thanks for your fix, @fnsne .  It's included in Version 2.11.1.
